src/OrchardCore.Themes/TheAdmin/Assets/scss/modules/admin/_switch.scss

```diff
@@ -20,7 +20,7 @@ $custom-control-indicator-disabled-bg: #e9ecef;
 $custom-control-description-disabled-color: #868e96;
 $white: white;
 $theme-colors: (
-    "$success": $success,
+    "success": $success,
```

Comment:
@Skrypt this was the change I had to make to get it to compile.

when I look at the css we were generating, there were instances of
`.alert-$success` in `TheAdmin.css`

Those gone now

Reply:
These switch are not maintained anymore. They are missing their javascript, I think it should probably be removed simply.